File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/04_compito_java_20150623/Appello.py
class Appello:
    def __init__(self, tipo, codice, docente, corso, giorno, mese, anno, ora):
        self._tipo = tipo
        self._codice = codice
        self._docente = docente
        self._corso = corso
        self._giorno = giorno
        self._mese = mese
        self._anno = anno
        self._ora = ora

    def get_ora_inizio(self):
        return self._ora

    def __str__(self):
        return str(self._codice) + "\t" + self._docente + "\t" + self._corso + "\t" + str(self._giorno) + "-" + str(self._mese) + "-" + str(self._anno) + "\t" + str(self._ora)

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/04_compito_java_2015_06_23/Appello.java
public class Appello {
    protected int codice;
    protected String docente;
    protected String corso;
    protected int giorno, mese, anno;
    protected int ora;

    public Appello(int codice, String docente, String corso, int giorno, int mese, int anno, int ora) {
        this.codice = codice;
        this.docente = docente;
        this.corso = corso;
        this.giorno = giorno;
        this.mese = mese;
        this.anno = anno;
        this.ora = ora;
    }

    public int getOra() {
        return ora;
    }
}